kratos/python_scripts/read_csv_table_utility.py

```diff
@@ -0,0 +1,79 @@
+import KratosMultiphysics as KM
+import csv
+
+class ReadCsvTableUtility:
+    r"""This class is used to retrieve a table from the specified parameters.
+
+    The following parameters can be specified:
+    |---------------|---------------------------------------------------|
+    | "name"        | The type of input (csv_table)                     |
+    |---------------|---------------------------------------------------|
+    | "file_name"   | The file name                                     |
+    |---------------|---------------------------------------------------|
+    | "delimiter"   | ","  comma                                        |
+    |               | ";"  semicolon                                    |
+    |               | "\t" tab                                          |
+    |               | " "  spaces                                       |
+    |               |      etc                                          |
+    |---------------|---------------------------------------------------|
+    | "skiprows"    | The number of rows to skip before reading data    |
+    |---------------|---------------------------------------------------|
+    | "table_id"    | If >-1 the input table will be stored in the      |
+    |               | model part                                        |
+    |---------------|---------------------------------------------------|
+    | "na_replace"  | The value to apply when N/A is read               |
+    |---------------|---------------------------------------------------|
+    """
+
+    def __init__(self, settings):
+        """Constructor of the csv table reader: validate the parameters.
+
+        Keyword arguments:
+        self -- It signifies an instance of the class.
+        settings -- Kratos parameters containing solver settings.
+        """
+        default_settings =  KM.Parameters("""{
+            "name"       : "csv_table",
+            "file_name"  : "",
+            "delimiter"  : ",",
+            "skiprows"   : 0,
+            "table_id"   : -1,
+            "na_replace" : 0.0
+        }""")
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        self.file_name = settings["file_name"].GetString()
+        self.delimiter = settings["delimiter"].GetString()
+        self.skiprows = settings["skiprows"].GetInt()
+        self.table_id = settings["table_id"].GetInt()
+        self.na_replace = settings["na_replace"].GetDouble()
+
+    def Read(self, model_part):
```

Comment:
```suggestion
    def Read(self, model_part = None):
```
So you don't need to pass the model part if the table is not to be stored.